server/scripts/create-database.ts

```diff
@@ -0,0 +1,51 @@
+import * as path from 'path';
+import * as dotenv from 'dotenv';
+import * as fs from 'fs';
+import { exec } from 'child_process';
+
+const createDatabase = (envPath: string): void => {
+  const result = dotenv.config({ path: envPath });
+
+  if (result.error) {
+    throw result.error;
+  }
+
+  exec('command -v createdb', (err, stdout, stderr) => {
+    if (err) {
+      console.error(err);
+      return;
+    }
+
+    const createdb =
+      `PGPASSWORD=${process.env.PG_PASS} createdb ` +
+      `-h ${process.env.PG_HOST} ` +
+      `-p ${process.env.PG_PORT} ` +
```

Comment:
Shouldn't there be defaults? `process.env.PG_HOST || 5432` ?

Reply:
Ah yes, missed adding. Will update.